front/src/app/components/forms/commons/AgencySelector.tsx

```diff
@@ -96,17 +99,27 @@ export const AgencySelector = ({
   const userError = !!(touched && error);
   const showError: boolean = userError || loadingError;
 
+  const agencyKindFilter = (
+    kind: AgencyKind,
+  ): kind is AllowedAgencyKindToAdd => {
+    if (shouldLockToPeAgencies) return kind === "pole-emploi";
+    if (shouldFilterDelegationPrescriptionAgencyKind)
+      return (
+        kind !== "autre" &&
+        kind !== "cci" &&
+        kind !== "operateur-cep" &&
+        kind !== "immersion-facile"
+      );
```

Comment:
on peut se réunir ça dans un tableau :

```suggestion
     const notFitForDelegationAgencyKind = ["autre", "cci", "operateur-cep", "immersion-facile"]
       [...]
      return (
        !notFitForDelegationAgencyKind.includes(kind)
      );
```

Reply:
Et pour éviter la double négation ca serait peut-être mieux de faire l'inverse :

```ts
const fitForDelegationAgencyKind: AgencyKind[] = ["pole-emploi", "mission-locale", "etc..."]

return fitForDelegationAgencyKind.includes(kind) 
```